.github/workflows/update.yml

```
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       GITHUB_TOKEN: ${{ secrets.REPO_GITHUB_TOKEN }}
```

We don't need this token as the script and data are now in the same repository.

```suggestion
      GITHUB_TOKEN: ${{ secrets.REPO_GITHUB_TOKEN }}
```